foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/internal/jpa/jpql/EclipseLinkSemanticValidatorHelper.java

```diff
@@ -378,6 +379,19 @@ public boolean isCollectionMapping(Object mapping) {
         }
     }
 
+    /**
+     * {@inheritDoc}
```

Comment:
I'd probably just rely on the `@Override` - can be a matter of taste

Reply:
Sounds good, removed